code/datums/hud/human.dm

```diff
@@ -958,3 +958,10 @@
 			var/datum/hud/human/H = src.hud
 			H.update_status_effects()
 		return
+
+mob/proc/rest()
+	.= 0
+mob/living/carbon/human/rest() //jesus fucking christ how do we not have a rest proc already, like what the fuck anyway now we do, you better use it
```

Comment:
Can probably get rid of the comment on this; I know we have plenty of examples of self-sass in the code, but we can move those to PR comments now. :)